changelog/better-vtemplates.dd

```diff
@@ -0,0 +1,14 @@
+Template usage diagnostics via `-vtemplates` has been improved.
+
+Every template having at least one instantiation is now printed using standard
+compiler diagnostics formatting for easy navigation to the point of its
+declaration.
+
+All messages of this kind are sorted by descending unique instantiation count.
+
+If the flag argument `list-instances` is fed as
+
+    `-vtemplates=list-instances`
+
+the location for every instance of each instantiated template is printed along
+with information telling whether the instance was implicit or explicit.
```

Comment:
```suggestion
If the flag argument `list-instances` is passed (such as `-vtemplates=list-instances`),
each location where a template was instantiated is printed along with the reason for
the instantiation (implicit or explicit).
```

Formatting looked weird and a terminating newline was missing, but then I tried to rephrase this to be (IMO) a bit more readable. Take it or leave it, as long as the formatting is fixed.

Also, perhaps add an explanation / example of what an implicit instantiation as opposed to an explicit one.

Reply:
Thanks. Done. Added a sample file aswell.